src/python/platforms/android/settings.py

```diff
@@ -194,3 +228,15 @@ def set_database_setting(database_path, table, key, value):
   sql_command_string = ('"UPDATE %s SET value=\'%s\' WHERE name=\'%s\'"') % (
       table, str(value), key)
   adb.run_shell_command(['sqlite3', database_path, sql_command_string])
+
+
+def get_repo_prop_path():
+  """Download repo.prop and return path of it on local machine."""
+  symbols_directory = environment.get_value('SYMBOLS_DIR')
+  _, _, repro_filename = symbols_downloader.get_kernel_info()
```

Comment:
see comment below, instead of unused args in unrelated function
build_id, target = settings.get_kernel_info()
repro_filename = settings.get_symbols_filename('repro.prop') or some other helper function.